go/vt/mysqlctl/xtrabackupengine.go

```diff
@@ -300,6 +312,7 @@ func (be *XtrabackupEngine) backupFiles(
 		"--slave-info",
 		"--user=" + xtrabackupUser,
 		"--target-dir=" + params.Cnf.TmpDir,
+		"--extra-lsndir=" + tempDir,
```

Comment:
@rvrangel will this flag exist in "most" recent versions of `xtrabackup`?

Reply:
good question, [it is present since the 2.4 EOL version](https://docs.percona.com/percona-xtrabackup/2.4/xtrabackup_bin/xbk_option_reference.html#extra-lsndirdirectory), so any recent version should have it I imagine.

Reply:
Where will the temporary directory be created by default? Are we sure to have the necessary permissions?
I'm wondering why not use `params.Cnf.TmpDir` and just write the file there.

Reply:
it will default to the system's `$TMPDIR` (usually `/tmp`) which we should almost always have permissions, the idea was that we could just remove it once we are done. but if we want to create it under `params.Cnf.TmpDir`, seems like the `--target-dir` is mostly unused because we stream the backup to `STDOUT`.

we could also use that path directly and instead of creating the temporary subdirectory and we either delete the files separately (it is just `xtrabackup_info` and `xtrabackup_checkpoints`) or we could just leave them there as the result of the last `xtrabackup` taken. @deepthi do you have a preference?

Reply:
What are the pros and cons of leaving the files behind versus deleting them? The next backup would overwrite them anyway, right?
Beyond this one decisions, almost any of the options would work. It's probably a bit simpler to not create a new temporary directory and just use one of the existing ones.

Reply:
leaving them is mostly for investigation (if needed), I guess we can just not remove them so there is a copy in the temp folder of the files from the last backup attempt in case an operator wants to look at it, otherwise it will just get overwritten the next time a backup is taken. I will update the PR tomorrow